agenta-cli/agenta/__init__.py

```diff
@@ -7,6 +7,7 @@
     InFile,
     IntParam,
     MultipleChoiceParam,
+    MessagesParam,
```

Comment:
I would actually name it MessagesInput and not param

Reply:
Sounds good. Thank you for the suggestion!